src/models/analytics-panel.ts

```diff
@@ -59,6 +59,7 @@ const schema = Joi.object().keys({
   layout: layoutSchema.default([]),
   cards: Joi.array().items(Joi.object().keys({
     title: Joi.string().required().example('My widget'),
+    subTitle: Joi.string().allow('').optional(),
```

Comment:
shouldn't be better to provide also any example?

Reply:
Done